mappings/net/minecraft/block/AbstractBlock.mapping

```diff
@@ -9,19 +9,27 @@ CLASS net/minecraft/class_4970 net/minecraft/block/AbstractBlock
 	FIELD field_23161 randomTicks Z
 	FIELD field_23162 soundGroup Lnet/minecraft/class_2498;
 	FIELD field_23163 slipperiness F
+	FIELD field_23164 velocityMultiplier F
 	FIELD field_23165 slowDownMultiplier F
 	METHOD <init> (Lnet/minecraft/class_4970$class_2251;)V
 		ARG 1 settings
 	METHOD method_16841 getOffsetType ()Lnet/minecraft/class_4970$class_2250;
 	METHOD method_17454 createScreenHandlerFactory (Lnet/minecraft/class_2680;Lnet/minecraft/class_1937;Lnet/minecraft/class_2338;)Lnet/minecraft/class_3908;
 	METHOD method_19286 onProjectileHit (Lnet/minecraft/class_1937;Lnet/minecraft/class_2680;Lnet/minecraft/class_3965;Lnet/minecraft/class_1676;)V
 	METHOD method_22358 canBucketPlace (Lnet/minecraft/class_2680;Lnet/minecraft/class_3611;)Z
+	METHOD method_25959 getSidesShape (Lnet/minecraft/class_2680;Lnet/minecraft/class_1922;Lnet/minecraft/class_2338;)Lnet/minecraft/class_265;
+	METHOD method_26159 getVisualShape (Lnet/minecraft/class_2680;Lnet/minecraft/class_1922;Lnet/minecraft/class_2338;Lnet/minecraft/class_3726;)Lnet/minecraft/class_265;
+	METHOD method_26160 asBlock ()Lnet/minecraft/class_2248;
+	METHOD method_26161 hasBlockEntity ()Z
+	METHOD method_26162 getDropTableID ()Lnet/minecraft/class_2960;
+	METHOD method_26163 getMaterialColor ()Ljava/util/function/Function;
 	METHOD method_8389 asItem ()Lnet/minecraft/class_1792;
 	METHOD method_9498 hasComparatorOutput (Lnet/minecraft/class_2680;)Z
 	METHOD method_9505 getOpacity (Lnet/minecraft/class_2680;Lnet/minecraft/class_1922;Lnet/minecraft/class_2338;)I
 	METHOD method_9506 emitsRedstonePower (Lnet/minecraft/class_2680;)Z
 	METHOD method_9514 scheduledTick (Lnet/minecraft/class_2680;Lnet/minecraft/class_3218;Lnet/minecraft/class_2338;Ljava/util/Random;)V
 	METHOD method_9516 canPathfindThrough (Lnet/minecraft/class_2680;Lnet/minecraft/class_1922;Lnet/minecraft/class_2338;Lnet/minecraft/class_10;)Z
+	METHOD method_9517 prepare (Lnet/minecraft/class_2680;Lnet/minecraft/class_1936;Lnet/minecraft/class_2338;I)V
```

Comment:
what does this do...

Reply:
Absolutely nothing! It’s called in `world#setBlockState` and `PistonBlock#move`, has no default impl, and no overrides.

Reply:
Interesting, looks good then.